article-store/events/tests/test_event_utils.py

```diff
@@ -9,15 +9,14 @@
 
 
 def test_can_get_base_message():
-    base_msg = create_message(msg_type='foo.test', identifier='someId', data={})
+    run_id = '2753d3ee-63de-11e8-9add-0242ac130008'
+    base_msg = create_message(msg_type='foo.test', run_id=run_id)
+
     assert base_msg
     assert len(base_msg['eventId'])
     assert base_msg['happenedAt']
     assert base_msg['type'] == 'foo.test'
-    assert base_msg['data'] == {}
-    assert base_msg['aggregate']['service'] == 'article-store'
-    assert base_msg['aggregate']['name'] == 'article-version'
-    assert base_msg['aggregate']['identifier'] == 'someId'
+    assert base_msg['runId'] == run_id
```

Comment:
Yeah, haven't been bothering with tests anywhere. #movefast #breakthings

Reply:
I have not been overdoing them, but having some present but failing was worth the fix.